src/main/java/de/adesso/service/FileTransfer.java

```diff
@@ -32,37 +23,17 @@ public FileTransfer(ConfigService configService) {
     void copyFile(File source, File dest) {
         try {
             if (source.lastModified() != dest.lastModified()) {
```

Comment:
Clean Code beschreibt optimal wartbaren Code als ein Buch das sich von oben bis unten liest.
Das auslagern von bool checks in eigene Methoden fördert dies enorm weil aus einem solchen Statement die Bedeutung direkt ersichtlich wird. 

Als ich den Vergleich gerade direkt demonstrieren wollte, ist mir aufgefallen, dass ich auf Anhieb gar nicht verstehe worauf genau geprüft wird. Trotz Vermutung, dass es der Check eindeutig ist, bin ich mir nicht sicher was genau erwartet wird. Dies sollte vermieden werden.

`if(source.lastModified() != dest.lastModified())` vs. `if(sourceFileIsModified)` oder etwas völlig anderes?